src/components/invite-dialog/index.tsx

```diff
@@ -88,6 +95,20 @@ export class InviteDialog extends React.Component<Properties, State> {
               {this.state.copyText}
             </button>
           </div>
+          <div className={c('remaining-invite')}>
+            <IconGift1
+              className={classNames({
```

Comment:
Rather than use this same classname twice you can put it on the parent element. Also, to be a touch more consistent with bem class naming you'd use the modifier.
```
<div className={classNames(c('remaining-invite'), {[c('remaining-invite', 'none')]: this.getInvitesRemaining() === 0})}>
```

Reply:
This worked in parent:
```js
className={
  classNames(c('remaining-invite'), {[c('no-invite-left')]: this.getInvitesRemaining() === 0})
}

But i don't fully understand how. I'll read up on `bem` & conditionally joining classnames more.
```

Reply:
So, your version would end up with a div that has classes:

`invite-dialog__remaining-invite invite-dialog__no-invite-left`

What you want is something like:

`invite-dialog__remaining-invite invite-dialog__remaining-invite--none`

That is, it's always the element `remaining-invite` and, in this case, it's also in the state `none`.